remoting/subsystem/src/main/java/org/jboss/as/remoting/Namespace.java

```diff
@@ -39,12 +39,13 @@
     REMOTING_1_2("urn:jboss:domain:remoting:1.2"),
     REMOTING_2_0("urn:jboss:domain:remoting:2.0"),
     REMOTING_3_0("urn:jboss:domain:remoting:3.0"),
+    REMOTING_3_1("urn:jboss:domain:remoting:3.1")
     ;
 
     /**
      * The current namespace version.
      */
-    public static final Namespace CURRENT = REMOTING_3_0;
+    public static final Namespace CURRENT = REMOTING_3_1;
```

Comment:
Yes. Schema version bumps should jump the same degree as the release that contains them. So major bumps in majors.

This ensures that any branches created from those release tags can add new schema versions without concern about conflicting with a schema that master or the branch from a later release has added.